.github/workflows/build_and_test.yml

```diff
@@ -42,3 +42,32 @@ jobs:
             --packages-up-to obstacle_stop_planner_refine \
             --event-handlers console_cohesion+ \
             --return-code-on-test-failure
+
+  code-metrics:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v2
+
+      - name: Measure code metrics
+        run: |
+          git clone https://github.com/terryyin/lizard.git
+          mkdir public
+          python3 lizard/lizard.py obstacle_stop_planner_refine/ -l cpp -x"*test*" \
+            --CCN 20 --length 200 --arguments 6 \
+            --html > public/lizard_only_wo_test.html || true
```

Comment:
btw, what does `only_wo_test` mean? :thinking:

Reply:
It might mean `lizard only` + `without test`. Parsing problem :smile: 
IMHO `code_metrics.html` or something might be more reasonable file name (and url).

Reply:
@kmiya Thanks! URL has changed to <https://tier4.github.io/obstacle_stop_planner_refine/code-metrics.html>.